build/Dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM ubuntu:latest
+
+# Python and gitsetup
+RUN apt-get update && apt-get install -y git && apt-get install -y python3 && apt-get install -y python3-pip
+
+# Clone repo
+RUN git clone https://github.com/dappnode/DAppNodeDocs.git && cd DAppNodeDocs
```

Comment:
Instead of cloning, you can just add the source from the same dir
```
COPY . .
```

Reply:
Also, you should add the source as the very last step since it's the most likely thing to change